dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -1569,6 +1569,10 @@ def test_diff(self):
             }
         }
         profile_diff = profiler1.diff(profiler2)
+        try:
+            json.dumps(profile_diff)
+        except TypeError:
+            self.fail('Object of type int64 is not JSON serializable in the diff')
```

Comment:
Instead of specifying the error that occurred before, it could be different. We could instead explain why it is an issue, the `Diff Profile is not JSON serializable`, we could also add in the exception itself:
```python
...
except TypeError as e:
            self.fail('<EXPLAINATION>. Exception raised: {}'.format(str(e)))
```
Then we can say the problem and what happened.